pyphare/pyphare/pharesee/hierarchy/patch.py

```
@@ -37,8 +37,15 @@ def __str__(self):
         return self.__str__()
 
+    def __iter__(self):
+        return self.patch_datas.items().__iter__()
+
     def __getitem__(self, key):
-        return self.patch_datas[key]
+        try:
+            return self.patch_datas[key]
+        except KeyError as e:
+            print(key, "not in", self.patch_datas.keys())
+            raise e
```

Ruff (0.15.4)

[warning] Use `raise` without specifying exception name

Remove exception name

(TRY201)